example/cypress/integration/logging-via-request-spec.js

```diff
@@ -4,6 +4,17 @@ describe('Logging in using XHR request', function() {
   const username = 'jane.lane';
   const password = 'password123';
 
+  before(() => {
+    // start recording
+    cy.recordHar();
```

Comment:
Would be nice to have ability to pause the har recording in the mid-stream.

Reply:
Hmm... Yep, we can try to implement that.

Reply:
Moved to #27